### DISCUSSION TITLE
The 1.0.3 ZIP version contains an outdated file as per 258f714 (Users.php): therefore the issue persists (solved by manual replacement)

### DISCUSSION THREAD

**Discussion by morphinestyle** [akeeba/panopticon#255] [Q&A (Community Support)] [answered]
The file https://github.com/akeeba/panopticon/blob/main/src/Controller/Users.php that resolves 258f714 does not match the one contained in the ZIP version of the version 1.0.3 downloadable package.

So whether you update or install from scratch you still get an error when accessing the profile page.

I solved it by downloading and replacing the Users.php file individually.

**Comment by nikosdion** [comment, marked as answer]
Not a bug, and the reason why I ask you to **please** not label issues and discussions yourself. Let me do the triaging.

If you actually read the Git log you will see that commit https://github.com/akeeba/panopticon/commit/258f7141e34e06a4710749f3b9324a5c97d2c8a2 was made _after_ the release of 1.0.3. So, yes, I did not travel back in time in violation of the Second Law of Thermodynamics to replace the file in a release I made a few hours before making that commit 😉

**Reply by morphinestyle** [reply]
Honestly I have never marked them as BUG, ​​I think some setting somewhere sets the label automatically.
I'll try to figure out where and what.

**Reply by nikosdion** [reply]
Darn it, I think GitHub automations are trolling us. Sigh.
